### ISSUE TITLE
[IntelliJ-ThirdParty][A11yMAS][Accessibility][NVDA][MAS4.2.1][IntelliJ][Wipro]:NVDA is not announicng appropriately the Static Text  present in Settings Window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1503]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Navigate and select Settings Menu item under File menu.
3. In Settings dialog  box, navigate and select Plugins, and then Select Browse repositories  button.
4. In Settings Window ,Press "Ctrl+Tab" then press tab  to access the static text then observe the NVDA Behavior this issue is observed entire static text present in window.

**Expected Result:**
**NVDA Behavior:** NVDA Should announce the static text "These plugins allow Java developers and Azure HDInsight users to easily create, develop, configure, test, and deploy highly available and scalable Java web apps and Spark/Hadoop jobs to Azure from IntelliJ on all supported platforms.
Azure Web App Workflow: You can run your web applications on Azure Web App with One-Click experience using Azure Toolkits for IntelliJ.
Azure Container Workflow: You can dockerize and run your web application on Azure Web App (Linux) via Azure Container Registry.
Azure Explorer: You can view and manage your cloud resources on Azure with Azure Explorer in Azure Toolkits for IntelliJ.
Azure HDInsight: Use Azure HDInsight tool to submit Spark jobs to HDInsight cluster, monitor and debug Spark or Hadoop Hive jobs easily"  in Settings window appropriately

**Actual Result:**
**NVDA Behavior:** NVDA is announicng as 
Version: 0.1
edit  text/html
Version: 0.1
tab
edit  text/html
blank
**Jaws Behavior:** Jaws is announcing as expected
Version: 3.3.0-2017.3
Use your reading keys to read the text.
text/html read only edit

These plugins allow Java developers and Azure HDInsight users to easily create, develop, configure, test, and deploy highly available and scalable Java web apps and Spark/Hadoop jobs to Azure from IntelliJ on all supported platforms.
Azure Web App Workflow: You can run your web applications on Azure Web App with One-Click experience using Azure Toolkits for IntelliJ.
Azure Container Workflow: You can dockerize and run your web application on Azure Web App (Linux) via Azure Container Registry.
Azure Explorer: You can view and manage your cloud resources on Azure with Azure Explorer in Azure Toolkits for IntelliJ.
Azure HDInsight: Use Azure HDInsight tool to submit Spark jobs to HDInsight cluster, monitor and debug Spark or Hadoop Hive jobs easily.
Use your reading keys to read the text.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)

![static text in settings](https://user-images.githubusercontent.com/22956130/36142196-5e679870-10cd-11e8-9314-09e1ac88ad01.jpg)
![plugins-static](https://user-images.githubusercontent.com/22956130/36142148-37cf714c-10cd-11e8-976f-e54cbe8a1a37.jpg)

**Comment by Eskibear** [comment]
Content is provided by IntelliJ IDEA and it's marketplace, it's out of the toolkit's control.

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by Eskibear** [comment]
And it's out of the toolkit's scope.

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749939
